src/OrchardCore.Modules/OrchardCore.PublishLater/Drivers/PublishLaterPartDisplayDriver.cs

```diff
@@ -37,10 +37,9 @@ public override IDisplayResult Display(PublishLaterPart part, BuildPartDisplayCo
             .Location("SummaryAdmin", "Meta:25");
         }
 
-        public override IDisplayResult Edit(PublishLaterPart part)
+        public override IDisplayResult Edit(PublishLaterPart part, BuildPartEditorContext context)
         {
-            return Initialize<PublishLaterPartViewModel>(
-                $"{nameof(PublishLaterPart)}_Edit",
+            return Initialize<PublishLaterPartViewModel>(GetEditorShapeType(context),
```

Comment:
Cool :).

Reply:
This allows for custom editors to be set in the part settings.

Reply:
Leaned something, thanks!